modules/eco-poison-pill-about-watchdog.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-watchdog-devices_{context}"]
+= About watchdog devices
+
+Watchdog devices are of the following categories:
+
+* Independently powered hardware devices 
+* Hardware devices that share power with the hosts they control
+* Virtual devices implemented in software (or softdog)
```

Comment:
If softdog is a linux kernel component, then I think you should wrap it in backticks. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Also, if you are explicitly referring to devices implemented in `softdog`, no need to say "software (or softdog)".

```suggestion
* Virtual devices implemented in `softdog`
```